src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = cleanWhiteSpace(description);
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void mark() {
+        this.isDone = true;
+    }
+
+    public void unmark() {
+        this.isDone = false;
+    }
+
+    public String cleanWhiteSpace(String word) {
```

Comment:
Maybe it could be easier to using java's .trim() method? Nonetheless, great work in trying to clean up the whitespaces!